lib/load.php

```diff
@@ -139,78 +139,43 @@ function gutenberg_is_experiment_enabled( $name ) {
 // Fonts API / Font Face.
 remove_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' ); // Turns off WordPress 6.0's stopgap handler.
 
+// Loads the Font Library.
+require __DIR__ . '/experimental/fonts/font-library/class-wp-font-collection.php';
+require __DIR__ . '/experimental/fonts/font-library/class-wp-font-library.php';
+require __DIR__ . '/experimental/fonts/font-library/class-wp-font-family-utils.php';
+require __DIR__ . '/experimental/fonts/font-library/class-wp-font-family.php';
+require __DIR__ . '/experimental/fonts/font-library/class-wp-rest-font-families-controller.php';
+require __DIR__ . '/experimental/fonts/font-library/class-wp-rest-font-collections-controller.php';
+require __DIR__ . '/experimental/fonts/font-library/class-wp-rest-autosave-font-families-controller.php';
+require __DIR__ . '/experimental/fonts/font-library/font-library.php';
+
+// Load the Font Face and Font Face Resolver, if not already loaded by WordPress Core.
+if ( ! class_exists( 'WP_Font_Face' ) ) {
```

Comment:
@matiasbenedetto this check needs to stay as Core loads these files. If the website has < WP 6.4.0, then the files are loaded.